both/collections/questions.js

```diff
@@ -0,0 +1,91 @@
+Questions = new Mongo.Collection('questions');
+
+QuestionSchema = new SimpleSchema({
+    id: {
+        type: String,
+        optional: false,
+        unique: true
+    },
+    questionType: {
```

Comment:
This field should probably have some `allowedValues` so that it is rendered in a select element. E.g. we only want to allow question types that we support, such as "multiple choice".